packages/react-button/src/components/Button/Button.styles.ts

```diff
@@ -0,0 +1,192 @@
+        borderColor: 'var(--button-pressed-borderColor, var(--button-hovered-borderColor, var(--button-borderColor)))',
+        boxShadow: 'var(--button-pressed-boxShadow, var(--button-hovered-boxShadow, var(--button-boxShadow)))',
+
+        transform: 'var(--button-pressed-transform)',
+        transition: 'var(--button-pressed-transition)',
+
+        '.ms-Button-icon': {
+          color: 'var(--button-pressed-iconColor, var(--button-iconColor))',
+        },
+      },
+    },
+  ],
+
+  icon: [
+    'ms-Button-icon',
+    {
+      color: 'var(--button-iconColor)',
+      display: 'flex',
+      alignItems: 'center',
+      justifyContent: 'center',
+      width: 'var(--button-iconSize)',
+      height: 'var(--button-iconSize)',
+      fontSize: 'var(--button-iconSize, inherit)',
+      fontWeight: 'normal',
+    },
+  ],
+
+  _primary: {
+    '--button-background': 'var(--accent-background)',
+    '--button-borderColor': 'var(--accent-borderColor)',
+    '--button-contentColor': 'var(--accent-contentColor)',
+    '--button-iconColor': 'var(--accent-iconColor)',
+
+    '--button-hovered-background': 'var(--accent-hovered-background)',
+    '--button-hovered-borderColor': 'var(--accent-hovered-borderColor)',
+    '--button-hovered-contentColor': 'var(--accent-hovered-contentColor)',
+    '--button-hovered-iconColor': 'var(--accent-hovered-iconColor)',
+
+    '--button-focused-background': 'var(--accent-focused-background)',
+    '--button-focused-borderColor': 'var(--accent-focused-borderColor)',
+    '--button-focused-contentColor': 'var(--accent-focused-contentColor)',
+    '--button-focused-iconColor': 'var(--accent-focused-iconColor)',
+
+    '--button-pressed-background': 'var(--accent-pressed-background)',
+    '--button-pressed-borderColor': 'var(--accent-pressed-borderColor)',
+    '--button-pressed-contentColor': 'var(--accent-pressed-contentColor)',
+    '--button-pressed-iconColor': 'var(--accent-pressed-iconColor)',
+
+    '--button-disabled-background': 'var(--accent-disabled-background)',
+    '--button-disabled-borderColor': 'var(--accent-disabled-borderColor)',
+    '--button-disabled-contentColor': 'var(--accent-disabled-contentColor)',
+    '--button-disabled-iconColor': 'var(--accent-disabled-iconColor)',
+  },
+
+  _iconOnly: {
+    '--button-width': 'var(--button-height, var(--button-minHeight))',
+    '--button-padding': 0,
+    paddingLeft: 0, // 'var(--button-padding)',
+    paddingRight: 0, // 'var(--button-padding)',
+    paddingTop: 0, // 'var(--button-padding)',
+    paddingBottom: 0, // 'var(--button-padding)',
+  },
+
+  _circular: {
+    '--button-borderRadius': 50000,
+  },
+
+  _fluid: {
+    width: '100%',
+    maxWidth: '100%',
+  },
+
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  _size_smallest: {
+    '--button-minHeight': 'var(--button-size-smallest)',
+  },
+
+  // eslint-disable-next-line @typescript-eslint/naming-convention
```

Comment:
hm can the rule be updated to support underscore prefix?

Reply:
possibly, or we disable it for the file.